src/main/kotlin/org/rust/lang/core/psi/impl/RustRawStringLiteralImpl.kt

```diff
@@ -1,12 +1,47 @@
 package org.rust.lang.core.psi.impl
 
 import com.intellij.psi.tree.IElementType
-import org.rust.lang.core.psi.RustLiteral
 import org.rust.lang.core.psi.RustLiteralTokenType
 
-class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustLiteral.Text(type, text) {
+class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustTextLiteralImplBase(type, text) {
+    override val value: String?
+        get() = valueString
+
     override fun toString(): String = "RustRawStringLiteralImpl($tokenType)"
 
+    private var hashes = 0
```

Comment:
Well, this means that I will have to calculate the same thing twice. I'll try to refactor offset computation to not use abstract methods etc instead.